### PR TITLE
Feat/responsive

### PR DESCRIPTION
**Pull request by LukyVj**
**What project are you opening a pull request for?**
- react-instantsearch (use v2 base)
- instantsearch.js (use develop base)

**Summary**

**Result**

### CODE REVIEW

File: docgen/layouts/common/meta.pug [hunk 1]
```diff
@@ -7,7 +7,7 @@ html
     meta(content='width=device-width,initial-scale=1.0,minimum-scale=1.0,maximum-scale=1.0,user-scalable=no', name='viewport')
     link(rel='icon', href="assets/img/react-instant-favicon.png")
     meta(content='IE=edge,chrome=1', http-equiv='X-UA-Compatible')
-    meta(content='react-instantsearch - Boost search on your React apps with Algolia', name='description')
+    meta(content='React-instantsearch - Boost search on your React apps with Algolia', name='description')
```
Comment (vvo) [inline docgen/layouts/common/meta.pug]: It's react-instantsearch

Comment (LukyVj) [inline docgen/layouts/common/meta.pug]: oops

**Comment by LukyVj** [comment]
I keep pushing stuff

**Comment by vvo** [comment]
Reopen pull request when done then?

**Comment by LukyVj** [comment]
yup ok

**Comment by LukyVj** [comment]
ping @vvo

**Comment by vvo** [comment]
Mergin'

**Pull request merged**